### ISSUE TITLE
Where did it go?

### ISSUE THREAD

**Issue by spinningarrow** [Try-Git/try_git#24]
https://try.github.io/levels/1/challenges/1 leads to a 404.

**Comment by aug25** [comment]
same for me, cannot TRY GIT

**Comment by eallenOP** [comment]
Yeah, where did it go? My students are trying to find it as evidence of independent learning, and they are now very confused!

**Comment by purvikadutt** [comment]
Same for me!  It's gone

**Comment by caztillo** [comment]
:(

**Comment by Zikoel** [comment]
It was removed without any informations?

**Comment by vinicius-stutz** [comment]
If I'm not mistaken, the previous try git was created in partnership with CodeSchool. Upon entering the CodeSchool website, we are redirected to the Pluralsight website; which leads me to believe that CodeSchool was acquired by Pluralsight. Maybe that was the reason for removing the previous try git page and even their respective repositories on GitHub.

**Comment by Paurian** [comment]
Thanks for details, vinicius-stutz. The command prompt functionality did look much like CodeSchool's work. Too bad Pluralsight didn't take that feature and run with it - they have all CodeSchool's videos, but the last time I checked, the functional code tests between the videos were gone.
